scripts/generate_export.py

```diff
@@ -451,11 +455,28 @@ def sequences_overlap(true_seq, pred_seq):
             ].apply(lambda x: x.ffill().bfill())
             output_df = output_df[col_order]
 
+            for cs in complete_submissions:
+                complete_filenames.append(str(cs.input_filename))
+
+            complete_filenames_df = pd.DataFrame(complete_filenames, columns=["filename"])
+        
+            for cs in complete_submissions:
+                sub_job = (IndicoClient.call(SubmissionResult(cs.id, wait=True)))
```

Comment:
These calls can be made with the IndicoWrapper object: https://github.com/IndicoDataSolutions/solutions_toolkit/blob/main/solutions_toolkit/indico_wrapper/indico_wrapper.py

get_submission_results and get_storage_object